roles/openshift_health_checker/openshift_checks/package_availability.py

```diff
@@ -0,0 +1,69 @@
+from openshift_checks import OpenShiftCheck, OpenShiftCheckException
+from openshift_checks.mixins import NotContainerized
+
+
+class PackageAvailability(NotContainerized, OpenShiftCheck):
+    """Check that required RPM packages are available."""
+
+    name = "package_availability"
+    tags = ["preflight"]
+
+    def run(self, tmp, task_vars):
+        try:
+            rpm_prefix = task_vars["openshift"]["common"]["service_type"]
+        except (KeyError, TypeError):
+            raise OpenShiftCheckException("'openshift.common.service_type' is undefined")
```

Comment:
I'm wondering if there is a better way to handle this rather than just pulling the value from the task_vars...  

I think it might be better to define an explicit interface for the checks, that way we are not bleeding the variable datastructures all the way down into the module. Especially as we start moving away from openshift_facts, I worry that this will cause some problems down the line.